eos-update-server/docs/eos-update-server.conf.5

```diff
@@ -41,6 +49,21 @@ Whether to advertise and distribute updates over the local network. This may be
 \fItrue\fP or \fIfalse\fP. If \fItrue\fP, \fBeos\-update\-server\fP(8) and
 \fBeos\-updater\-avahi\fP(8) are enabled; otherwise, they will both refuse to
 advertise or distribute updates.
+\"
+.SH [Repository 0–9] SECTION OPTIONS
```

Comment:
I decided to go with numbering in the root path (rather than a string-based name) because the root path needs to be put in the DNS-SD records, and hence needs to be as small as possible. As you noticed, there is no ordering (or contiguity) enforced.

I’ll see if I can improve the documentation. I realise `[Repository 0–9]` is not particularly clear. I could go with `[Repository 0-9+]` (regex-style)?